### ISSUE TITLE
SkeletonView with pagination using TableView

### ISSUE THREAD

**Issue by MamtaKukreja** [Juanpe/SkeletonView#247]
### What did you do?

Show skeleton on few labels of cell, on every API call and hide it once API call is complete

### What did you expect to happen?

Expected to work smoothly on pagination too

### What happened instead?

If a slow scroll made pagination to occur, it just shows a blank space instead of skeletoned cells, and if its a fast scroll, it shows the skeleton and then scrolls up by 5-6 records - which is not smooth at all.

### Steps to reproduce the behavior

Show Skeleton on API call
Hide it when API call is done
Scroll down the tableview until pagination occurs

### SkeletonView Environment

**SkeletonView version: pod 1.8.3**
**Xcode version: 11.2.1**
**Swift version: 5.0**

**Comment by Juanpe** [comment]
Hi @MamtaKukreja, when the skeleton is visible, the user interaction is disabled, so you shouldn't scroll the table view. If you can do it, we are talking about another issue. 

Please, could you confirm that you can scroll the table view when the skeleton view is active?

**Comment by MamtaKukreja** [comment]
Hi @Juanpe,
No, I cannot scroll the tableview when the skeleton is active. Once records are loaded from API in table and we scroll 20 rows, we call the API again to fetch next 20 records and show skeleton meanwhile. I am facing this issue in that scenario.

**Comment by Juanpe** [comment]
Now, I understand your problem. In my opinion, the skeleton should be used only for the first request. If the user is scrolling and the skeleton appears, it could be very intrusive.

Your scenario is very complex, I mean, the velocity of the scroll, the number of cells that we already have...Sorry, but for now I haven't included this scenario in the backlog :/

**Comment by MamtaKukreja** [comment]
Thanks for the quick response @Juanpe
